buku

```diff
@@ -3376,6 +3384,31 @@ def convert_bookmark_set(
             count += 1
 
         out += '</xbel>'
+    elif export_type == 'rss':
```

Comment:
Also [a test case](https://github.com/jarun/buku/blob/master/tests/test_buku.py#L842-L910) for this block should be added

Reply:
Done